src/util/codec/bytes.rs

```diff
@@ -193,7 +193,7 @@ pub fn decode_bytes(data: &mut BytesSlice, desc: bool) -> Result<Vec<u8>> {
         if padding.iter().any(|x| *x != pad_byte) {
```

Comment:
do we need to alloc extra u64 size for the decode_bytes?

Reply:
no need, decode_bytes() creates raw key which will not append ts.

Reply:
no need, seems that decode_bytes() will not reach and push the appended ts bytes